samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -0,0 +1,1487 @@
+#!/usr/bin/env bash
+: '
+Copyright (C) 2021 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the “License”);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an “AS IS” BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'
+#-------------------------------------------------------------------------
+set -e
+#set -x
+
+function help_function {
+  cat <<-EOF
+
+Automation for creating RHCOS/RHEL/CENTOS images to PowerVS Services. This is a wrapper for pvsadm tool.
+
+Usage:
+  ./convert-upload-images-powervs [ --rhel-url <url> | --rhcos-url <url> | --centos-url <url> ] --service-name  <service name> --region <bucket region> --cos-bucket <bucket name> --cos-resource-group <resource group> [ --cos-instance-name <cos instance name> ]
+
+Args:
+      --service-name string       A service name in cloud.ibm.com you have access(Mandatory)
+      --region string             Object store bucket region(Mandatory)
+      --cos-bucket                Object store bucket name(Mandatory)
+      --cos-resource-group        COS resource group(Mandatory)
+      --cos-instance-name         COS instance name(optional)
+      --centos-url  url           url pointing to the centos qcow2 image(optional)
+      --rhel-url  url             url pointing to the rhel qcow2 image(optional)
+      --rhcos  url                url pointing to the rhcos qcow2 image(optional)
+      --help                      help for upload
+
+EOF
+  exit 0
+}
+
+
+ARTIFACTS_VERSION="master"
+
+
+
+TF='./terraform'
+CLI_PATH='./ibmcloud'
+TMP_DIR=$(mktemp -d tmp.XXXXXXXXXXX)
+
+ARTIFACTS_DIR="automation"
+LOGFILE="$(date "+%Y%m%d%H%M%S")"
+GIT_URL="https://github.com/ocp-power-automation/infra"
+
+TRACE=0
+TF_TRACE=0
+FORCE_DESTROY=1
+
+NO_OF_RETRY=${NO_OF_RETRY:-"5"}
+SLEEP_TIME=10
+REBOOT_TIMEOUT=30
+
+PVSADM_VERSION="v0.1.1-alpha.2"
+IMAGE_SIZE="120"
+
+# Default Centos OVA image details
+COS_CENTOS_OVA_BUCKET_NAME='ocp4-prod-images'
+CENTOS_OVA_OBJECT_NAME='centos-02232021-250.ova.gz'
+CENTOS_OVA_IMAGE_NAME='centos-02232021-250'
+COS_CENTOS_OVA_BUCKET_REGION='us-south'
+
```

Comment:
Why do we need the above values ?

Reply:
Added this because of a confusion. Will add the option to use default images

Reply:
Removed this.